index.js

```diff
@@ -18,19 +19,20 @@ function useCustom(store, React, mapState, mapActions) {
     const newListener = { oldState: {} };
     newListener.run = mapState
       ? newState => {
-          const mappedState = mapState(newState);
-          if (mappedState !== newListener.oldState) {
-            newListener.oldState = mappedState;
-            originalHook(mappedState);
-          }
+        const mappedState = mapState(newState);
+        if (mappedState !== newListener.oldState) {
+          newListener.oldState = mappedState;
+          originalHook(mappedState);
```

Comment:
the `mappedState` argument is actually confusing as this value will never be used: it just needs to be a different reference each time
- see https://github.com/andregardi/use-global-hook/pull/59